pkgs/tools/misc/kak-lsp/default.nix

```diff
@@ -0,0 +1,25 @@
+{ lib, fetchFromGitHub, rustPlatform }:
```

Comment:
```suggestion
{stdenv, lib, fetchFromGitHub, rustPlatform }:
```

fix eval

Reply:
Ah, thanks! Completely missed that :sweat_smile:

Reply:
oops we need `darwin` too